okteto.yml

```diff
@@ -0,0 +1,45 @@
+name: node-compose-getting-started
+
+build:
+  api:
+    context: api
+    dockerfile: api/Dockerfile
+    image: okteto.dev/api
+
+  web:
+    context: web
+    dockerfile: web/Dockerfile
+    image: okteto.dev/web
+
+  web-dev:
+    context: web
+    target: dev
+    dockerfile: web/Dockerfile
+    image: okteto.dev/web-dev
```

Comment:
minor: you can remove `image` here and use `OKTETO_BUILD_WEB_DEV_IMAGE` in your dev container. That way, you always get the latest sha of the image.